bin/mini-langstream/README.md

```diff
@@ -0,0 +1,40 @@
+# mini-langstream
+
+## Get started
+
+### MacOS/Linux/WSL
+
+```
+curl -Ls "https://raw.githubusercontent.com/LangStream/langstream/main/bin/mini-langstream/get-mini-langstream.sh" | bash
```

Comment:
Can we make it available in the homebrew formula ?

Personally (maybe it is a matter of taste) I don't like to install stuff this way on my Mac, as there is no way of uninstalling things. With homebrew (and other managers) the whole lifecycle is handled (also the upgrades)

Reply:
yes this is for Linux users. Brew will come later